### ISSUE TITLE
twisted.mail.imap4.IMAP4Client.append docstring

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#11154]
|<img alt="synapsis's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| synapsis reported|
|-|-|
|Trac ID|trac#1648|
|Type|defect|
|Created|2006-04-17 15:32:50Z|

twisted.mail.imap4.IMAP4Client.append has a wrong docstring:

"""Add the given message to the currently selected mailbox"""

should be

"""Add the given message to the given mailbox"""

<details><summary>Searchable metadata</summary>

```
trac-id__1648 1648
type__defect defect
reporter__synapsis synapsis
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__mail mail
keywords__ 
time__1145287970000000 1145287970000000
changetime__1149461117000000 1149461117000000
version__ 
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set status to `closed`|
|-|-|

(In [17078]) Fix IMAP4Client.append docstring

Author: synapsis
Reviewer: exarkun
Fixes [#11154](https://github.com/twisted/twisted/issues/11154)

Append adds a message to a mailbox which you specify when calling it,
not the mailbox which has previously been selected.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
